### PR TITLE
Update ont-leox-lxt-010h-d.md

### PR DESCRIPTION
**Pull request by stich86**

### CODE REVIEW

**Comment by stich86** [comment]
Add photos of ONT LEOX LXT-010H-D and some other informations

**Review by simonebortolin — APPROVED** [review]

File: _ont/ont-leox-lxt-010h-d.md [hunk 1]
```diff
@@ -27,8 +27,49 @@ parent: LEOX
...
+
+{% include image.html file="ont-leox-lxt-010-h-d_top.jpg" alt="Leox LXT-010H-D Top" caption="Leox LXT-010H-D Top" %} 
+{% include image.html file="ont-leox-lxt-010-h-d_bottom.jpg" alt="Leox LXT-010H-D Bottom" caption="Leox LXT-010H-D Bottom" %}
+
 # Serial
 
-The ONT has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be accessed from the top surface.
+The ONT has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be accessed from the top surface (see red square).
+
+{% include image.html file="ont-leox-lxt-010-h-d_ttl" alt="Leox LXT-010H-D TTL Pinout" caption="Leox LXT-010H-D TTL Pinout" %}
```
Comment (simonebortolin) [inline _ont/ont-leox-lxt-010h-d.md:39]: Missing file extension

**Review by simonebortolin — CHANGES_REQUESTED** [review]

File: _ont/ont-leox-lxt-010h-d.md [hunk 2]
```diff
@@ -27,8 +27,49 @@ parent: LEOX
...
+This ONT supports dual boot. 
+
+`k0` and `r0` respectively contain the kernel and firmware of the first image, while `k1` and `r1` respectively contain the kernel and the firmware of the second one.
+
+## List firmware version
+
+- V3.3.2L6
+- V3.3.2L7
+
+{% include alert.html content="Before proceed on any modification, make a backup of files rtl8290b.data and europa.data from /var/config folder. These files include optical calibration of your ONT's laser, if you accidentally delete or  ruin them, your ONT will be unusable" alert="Note" icon="svg-warning" color="yellow" %}
```
Comment (simonebortolin) [inline _ont/ont-leox-lxt-010h-d.md:71]: Delete blank space before ruin

**Review by simonebortolin — CHANGES_REQUESTED** [review]

File: _ont/ont-leox-lxt-010h-d.md (hunk 1 above)
Comment (stich86) [reply _ont/ont-leox-lxt-010h-d.md]: fixed

File: _ont/ont-leox-lxt-010h-d.md (hunk 2 above)
Comment (stich86) [reply _ont/ont-leox-lxt-010h-d.md]: fixed

**Review by simonebortolin — APPROVED** [review]

**Pull request merged**